### PR TITLE
add processing for dcatus contactPoint

### PR DESCRIPTION
**Pull request by rshewitt**
related to [#4886](https://github.com/GSA/data.gov/issues/4886)

- add pointOfContact processing
- add eMailList processing
- add to fixtures

### CODE REVIEW

**Comment by rshewitt** [comment]
@btylerburton

File: test/translator/tc_iso19115_2_to_dcatus.rb [hunk 1]
```diff
@@ -66,4 +66,12 @@ def test_publisher
       expected = '{"@type":"org:Organization","name":"organization name"}'
       assert_equal(expected, res)
    end
+
+   def test_contact_point
+      dcatusNS = ADIWG::Mdtranslator::Writers::Dcat_us::ContactPoint
+      res = dcatusNS.build(@@intMetadata).target!
+
+      expected = '{"@type":"vcard:Contact","fn":"person name","hasEmail":"email@test.com"}'
```
Comment (btylerburton) [inline test/translator/tc_iso19115_2_to_dcatus.rb:74]: why do we expect `email@test.com` and not `test@gmail.com`?
should we only add that to a single node in `test/translator/testData/iso19115-2.xml`? is that meaningful?

Comment (rshewitt) [reply test/translator/tc_iso19115_2_to_dcatus.rb:74]: we're looking inside `gmd:pointOfContact` not `gmd:citation` within `gmd:MD_DataIdentification`. `email@test.com` is the email address for the first responsible party in `gmd:pointOfContact`

Comment (btylerburton) [reply test/translator/tc_iso19115_2_to_dcatus.rb:81]: I figured as much, thanks for the confirmation!

**Review by btylerburton — APPROVED** [review]

**Pull request merged**